packages/patternlab/styleguide/source/assets/js/modules/skipLinkTarget.js

```diff
@@ -33,25 +32,26 @@ document.querySelectorAll(".ma__figure__skip-link").forEach(link => {
       document.querySelector(location.hash).style.display = "none";
     }
   });
-});
-
 
-// For TOC skip link target
-const skipLinkTOC = document.querySelector(".ma__jump-links__skip-link");
-const skipLinkTOCTarget = document.querySelector("#ma-end-sticky-toc");
+  // For TOC skip link target
```

Comment:
When the `forEach` closes above, it causes the error you found.  So, here, we can wrap LL. 40-55 with
`if (skipLinkTOC) { }`.  Then, pages don't have TOC skip links, in other words, no TOC, wouldn't get the error.

I kept the code for these two different skip links together here, which is my bad, but thought it would be easier to manage since they have the same functionality with different targets.

Reply:
I added a check for both the TOC and the Target, as I noticed jump links have TOC but not the target, and this won't apply.

Reply:
Excellent!  Thank you for the change.